plugins/repositories/repositories_renderer.rb

```diff
@@ -25,13 +25,21 @@ def content(description)
 
     list do
       description.repositories.each do |p|
-        item "#{p.name}" do
-          puts "URI: #{p.url}"
-          puts "Alias: #{p.alias}"
-          puts "Enabled: #{p.enabled ? "Yes" : "No"}"
+        if p.name
+          item_name = p.name
+        else
+          item_name = "URI: #{p.url}"
+        end
+        item "#{item_name}" do
+          puts "URI: #{p.url}" if p.name
+          puts "Alias: #{p.alias}" if p.alias
+          puts "Distribution: #{p.distribution}" if p.distribution
+          puts "Components: #{p.components.join(", ")}" if p.components
+          puts "Enabled: #{p.enabled ? "Yes" : "No"}" if !p.enabled.nil?
           puts "Refresh: #{p.autorefresh ? "Yes" : "No"}" if p.autorefresh != nil
           puts "Priority: #{p.priority}" if p.priority != nil
-          puts "Package Manager: #{p.package_manager}"
+          puts "Type: #{p.type}"
+          puts "Package Manager: #{description.repositories.repository_system}"
```

Comment:
Do you really think we should show this again for each repository? That's a lot of redundancy and it's not what we're doing for services, for example.